utils/gov/formatData.ts

```diff
@@ -0,0 +1,177 @@
+// export function formatDate(voting_end_time: string): string {
+//     const date = new Date(voting_end_time);
+    
+//     const day = String(date.getUTCDate()).padStart(2, '0');
+//     const month = String(date.getUTCMonth() + 1).padStart(2, '0');
+//     const year = String(date.getUTCFullYear()).slice(-2);
+
+//     return `${day}/${month}/${year}`;
+// }
+export function formatDate(voting_end_time: string): string {
```

Comment:
this function is never used anywhere in the code

Reply:
Deleted this